rfcs/0-example.md

```diff
@@ -1,10 +1,15 @@
----
-author: Xuanwo <github@xuanwo.io>
-status: [draft, candidate, finished]
-updated_at: 2019-11-15
----
-
-# Proposal: <proposal name>
+- Author: (fill me in with `name <mail>`, e.g., Xuanwo <github@xuanwo.io>)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [beyondstorage/specs#0](https://github.com/beyondstorage/specs/issues/0)
+- Tracking Issue: [beyondstorage/go-storage#0](https://github.com/beyondstorage/go-storage/issues/0)
+- Updates: (delete this part if not applicable)
+  - [GSP-20](./20-abc): Deletes something
+- Updated By: (delete this part if not applicable)
+  - [GSP-10](./10-do-be-do-be-do): Adds something
+  - [GSP-1000](./1000-lalala): Deprecates this RFC
```

Comment:
Or put them in another section to keep the header clean?

Reply:
I support to put them in another section.